thunder/tests/test_core.py

```diff
@@ -1865,20 +1865,6 @@ def torch_to(a, memory_format):
         assert_close(torch_result, thunder_result, check_stride=True)
 
 
-def test_torch_tensor_to_return_type():
```

Comment:
OpInfos is always the preferred way of testing functionality. Thanks for updating the test cases `to_opinfo `!